dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.2.ebuild

```diff
@@ -1,38 +1,44 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-inherit unpacker cmake-multilib
+inherit cmake-multilib
 
-MY_P="deviceatlas-enterprise-c-${PV}"
+MY_P="deviceatlas-enterprise-c-${PV}_1"
```

Comment:
Are you sure that upstream will never release _2? If not, then I think it'd be better to name it `2.1.2_p1` and then `${PV/_p/_}`.